fixtures/stairville/mh-x25.json

```diff
@@ -0,0 +1,436 @@
+        },
+        {
+          "range": [24, 31],
+          "name": "Blackout while color change"
+        },
+        {
+          "range": [32, 39],
+          "name": "No Blackout while color change"
+        },
+        {
+          "range": [40, 47],
+          "name": "Blackout while gobo change"
+        },
+        {
+          "range": [48, 55],
+          "name": "No Blackout while gobo change"
+        },
+        {
+          "range": [56, 87],
+          "name": "Empty"
+        },
+        {
+          "range": [88, 95],
+          "name": "Blackout while movement"
+        },
+        {
+          "range": [96, 103],
+          "name": "Reset pan and tilt"
+        },
+        {
+          "range": [104, 111],
+          "name": "Empty"
+        },
+        {
+          "range": [112, 119],
+          "name": "Reset color wheel"
+        },
+        {
+          "range": [120, 127],
+          "name": "Reset gobo wheel"
+        },
+        {
+          "range": [128, 135],
+          "name": "Reset gobo rotation"
+        },
+        {
+          "range": [136, 151],
+          "name": "Empty"
+        },
+        {
+          "range": [152, 159],
+          "name": "Reset all channels"
+        },
+        {
+          "range": [160, 255],
+          "name": "Empty"
+        }
+      ]
+    },
+    "Programs": {
+      "type": "Effect",
+      "capabilities": [
+        {
+          "range": [0, 7],
+          "name": "Off"
+        },
+        {
+          "range": [8, 23],
+          "name": "Program 1"
+        },
+        {
+          "range": [24, 39],
+          "name": "Program 2"
+        },
+        {
+          "range": [40, 55],
+          "name": "Program 3"
+        },
+        {
+          "range": [56, 71],
+          "name": "Program 4"
+        },
+        {
+          "range": [72, 87],
+          "name": "Program 5"
+        },
+        {
+          "range": [88, 103],
+          "name": "Program 6"
+        },
+        {
+          "range": [104, 119],
+          "name": "Program 7"
+        },
+        {
+          "range": [120, 135],
+          "name": "Program 8"
+        },
+        {
+          "range": [136, 150],
+          "name": "Sound control 1"
+        },
+        {
+          "range": [151, 165],
+          "name": "Sound control 2"
+        },
+        {
+          "range": [166, 180],
+          "name": "Sound control 3"
+        },
+        {
+          "range": [181, 195],
+          "name": "Sound control 4"
+        },
+        {
+          "range": [196, 210],
+          "name": "Sound control 5"
+        },
+        {
+          "range": [211, 225],
+          "name": "Sound control 6"
+        },
+        {
+          "range": [226, 240],
+          "name": "Sound control 7"
+        },
+        {
+          "range": [241, 255],
+          "name": "Sound control 8"
+        }
+      ]
+    }
+  },
+  "modes": [
+    {
+      "name": "6-Channel",
```

Comment:
Make "Channel" lowercase. That's also why our tests didn't warn about a wrong channel number.